extensions/amp-story/1.0/_locales/en.json

```diff
@@ -229,7 +229,7 @@
   },
   "104": {
     "string": "reviews",
-    "description": "Button label prompting users to read about peoples assessments of an item."
+    "description": "Button label prompting users to read about peoples assessments of an item. It is not capitalized since it is after a number."
```

Comment:
Should we show an example of this? `It is not capitalized since it comes after a number (eg: "87 reviews")`